project/Dependencies.scala

```diff
@@ -54,7 +54,9 @@ object Dependencies {
     "org.scalatest" %% "scalatest" % "3.2.2" % "it,test",
     "org.scalamock" %% "scalamock" % "5.0.0" % "test",
     "org.scalatestplus" %% "scalacheck-1-14" % "3.2.2.0" % "test",
-    "org.scalacheck" %% "scalacheck" % "1.14.3" % "it,test"
+    "org.scalacheck" %% "scalacheck" % "1.14.3" % "it,test",
+    "com.softwaremill.diffx" %% "diffx-core" % "0.3.30" % "test",
+    "com.softwaremill.diffx" %% "diffx-scalatest" % "0.3.30" % "test"
```

Comment:
Don't we need to update nix dependencies?

Reply:
We do, I wanted buildkite to generate proper updates for me